basicview.cpp

```diff
@@ -59,22 +98,26 @@ BasicView::BasicView()
     connect(sshButton, &QAbstractButton::clicked, this, &BasicView::ssh);
     connect(dashboardButton, &QAbstractButton::clicked, this, &BasicView::dashboard);
     connect(advancedButton, &QAbstractButton::clicked, this, &BasicView::advanced);
+
 }
 
+
 static QString getPauseLabel(bool isPaused)
 {
     if (isPaused) {
         return "Unpause";
```

Comment:
This is going to change the button to text once it's paused. Will want to make this a icon as well.

Reply:
yeah I know unfortunately we dont have icon for it now in FA 5, I am gonna have it as is for now, till we get the FA working